.changes/introduce-run-return.md

```diff
@@ -0,0 +1,9 @@
+---
+tauri: 'minor:feat'
+tauri-runtime: 'minor:feat'
+tauri-runtime-wry: 'minor:feat'
+---
+
+Add `App::run_return` function. Contrary to `App::run`, this will **not** exit the process but instead return the requested exit-code. This allows the host app to perform further cleanup after Tauri has exited. `App::run_return` is only available on desktop platforms.
```

Comment:
```suggestion
Add `App::run_return` function. Contrary to `App::run`, this will **not** exit the process but instead return the requested exit-code. This allows the host app to perform further cleanup after Tauri has exited. `App::run_return` is not available on iOS and fallbacks to the regular `App::run` functionality.
```